astropy/visualization/wcsaxes/core.py

```diff
@@ -451,10 +455,19 @@ def draw(self, renderer, inframe=False):
 
         self._drawn = True
 
-    def set_xlabel(self, label, labelpad=1, **kwargs):
+    # The label=None, xlabel=None (and similarly for ylabel) is to ensure
+    # compatibility with Matplotlib 2.x (which has label) and 3.x (which has
+    # xlabel). While these are meant to be a single positional argument,
+    # Matplotlib internally sometimes specifies e.g. set_xlabel(xlabel=...).
+
+    def set_xlabel(self, label=None, xlabel=None, labelpad=1, **kwargs):
```

Comment:
This is very pedantic, but perhaps write the code to be consistent with the latest version of matplotlib. Would the following work?
```
def set_xlabel(xlabel=None, labelpad=1, **kwargs):
    if xlabel is None:
        xlabel = kwargs.pop('label', None)
```
Just in case: both the above and your change mean that passing in nothing will work (in that `None` gets passed down), unlike what was the case previously.